rts/Lua/LuaSyncedRead.cpp

```diff
@@ -8999,6 +9001,19 @@ int LuaSyncedRead::GetRadarErrorParams(lua_State* L)
 	return 3;
 }
 
+/***
+ *
+ * @function Spring.IsFeatureVisible
+ * @param featureID integer
+ * @return number?
+ */
+int LuaSyncedRead::IsFeatureVisible(lua_State* L)
+{
+	const CFeature* feature = ParseFeature(L, __func__, 1);
+	lua_pushboolean(L, feature != nullptr && LuaUtils::IsFeatureVisible(L, feature));
+	return 1;
```

Comment:
would be nice if this differentiated between "not visible" vs "such feature does not exist" for full-read handles, but it's also ok as-is already

Reply:
The established pattern is to return `nil` on invalid arguments.

Personally I think it would be a better pattern to return false and log an error, and instead provide a separate function that checks the validity of a feature ID for the unlikely case where that is required.

I'm happy with this implementation for now though.

Reply:
ditto